### PR TITLE
add os-climate subproject to sig-data-science

### LINKED ISSUE #35: Add OS-C subproject under sig-data-science
We nee to add the OS-C work as a sub-project under the data science sig. 

Update the following files according to the [new subprojects doc](https://github.com/open-services-group/community/blob/main/sig-data-science/how-to-create-a-subproject.md): 

- [ ] update sig-data-science/README.md
- [ ] update sigs.yaml

### PR DESCRIPTION
**Pull request by MichaelClifford**
closes #35 

This PR updates the sigs.yaml and sig-data-science/README.md to add the OS-Climate subproject. It also removes the subprojects that we decided against in our last sig meeting.

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -30,20 +30,22 @@ sigs:
...
-    - https://raw.githubusercontent.com/open-services-group/community/main/sig-data-science/OWNERS
-  - name: MLOps
-    owners:
-    - https://raw.githubusercontent.com/open-services-group/community/main/sig-data-science/OWNERS
-  - name: Monitoring
+  - name: OS-Climate
+    description: '[Described below](#os-climate)'
     owners:
-    - https://raw.githubusercontent.com/open-services-group/community/main/sig-data-science/OWNERS
+    - https://github.com/os-climate/aicoe-osc-demo/blob/master/OWNERS
```
Comment (MichaelClifford) [inline sigs.yaml:48]: @erikerlandson  are there other repos we should track here as being "owned" by this subproject?

Comment (erikerlandson) [reply sigs.yaml:48]: a few more:
https://github.com/os-climate/osc-trino-acl-dsl/blob/main/OWNERS
https://github.com/os-climate/osc-ingest-tools/blob/main/OWNERS
https://github.com/os-climate/data-platform-demo

Comment (erikerlandson) [reply sigs.yaml:48]: oops should be:
https://github.com/os-climate/data-platform-demo/blob/master/OWNERS

Comment (MichaelClifford) [reply sigs.yaml:48]: Thanks!

Comment (erikerlandson) [reply sigs.yaml:48]: One last question - are these supposed to be to raw-files, like:
https://raw.githubusercontent.com/os-climate/aicoe-osc-demo/master/OWNERS

Comment (MichaelClifford) [reply sigs.yaml:48]: good catch, yes they need to be the raw files. Thanks I'll make the change

**Review by erikerlandson — APPROVED** [review]

**Comment by MichaelClifford** [comment]
cc: @open-services-group/steering

**Comment by goern** [comment]
please run `podman run --rm --volume $(pwd):/workdir:Z quay.io/open-services-group/community-tooling:v0.1.0-dev` so that the markdown files get updated. thx

/lgtm

**Comment by MichaelClifford** [comment]
Thanks @goern update made :)

**Comment by erikerlandson** [comment]
if there are no additional changes, can @open-services-group/steering approve?

**Comment by MichaelClifford** [comment]
cc: @durandom @goern

**Comment by oindrillac** [comment]
> podman run --rm --volume $(pwd):/workdir:Z quay.io/open-services-group/community-tooling:v0.1.0-dev

We should also document this step in https://github.com/open-services-group/community/blob/main/sig-data-science/how-to-create-a-subproject.md

**Comment by MichaelClifford** [comment]
>> podman run --rm --volume $(pwd):/workdir:Z quay.io/open-services-group/community-tooling:v0.1.0-dev

> We should also document this step in https://github.com/open-services-group/community/blob/main/sig-data-science/how-to-create-a-subproject.md

@oindrillac we can do that, but for now, this step needs to be done for all PR's to this repo. So maybe it needs to be added to a "how to contribute" doc as well.

**Comment by tumido** [comment]
@oindrillac @MichaelClifford see and comment on #26 for ^

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**